app/views/templates/main_template.scala.html

```diff
@@ -71,7 +71,7 @@
     @mainContent
 }
 
-@govuk_wrapper(title = title,
+@govuk_wrapper(title = title + " " + messages("titlePattern.serviceName.govUk"),
```

Comment:
Not too much of a fan of this, might be cleaner to update the message to be
`{0} - Business Tax account - GOV.UK` that way here you could just do 
`title = messages("titlePattern.serviceName.govUk", title)`
without needing to use `+ " " +`

Reply:
done